src/pages/search.js

```diff
@@ -40,12 +52,87 @@ export default withSearch(({ search }) => {
     ? ` : ${query} (${totalHits} result${totalHits === 1 ? '' : 's'})`
     : ''
 
-  let searchEvent
+  const getSlug = item => {
+    if (!item.publishDate) {
+      return `${item.slug}`
+    }
+    return `/blog/${item.slug}`
+  }
+
+  const goToResult = selectedItem => {
```

Comment:
It amazes me Combobox doesn't allow choosing what we pass. @kevee do you know if there's a way to pass the `item.objectID` instead?